My6sense new adapter

my6sense opened the pull request:
<!--
Thank you for your pull request. Please make sure this PR is scoped to one change, and that any added or changed code includes tests with greater than 80% code coverage. See https://github.com/prebid/Prebid.js/blob/master/CONTRIBUTING.md#testing-prebidjs for documentation on testing Prebid.js.
-->

## Type of change
<!-- Remove items that don't apply and/or select an item by changing [ ] to [x] -->
- [ ] Bugfix
- [ ] Feature
- [X] New bidder adapter  <!--  IMPORTANT: if checking here, also submit your bidder params documentation here https://github.com/prebid/prebid.github.io/tree/master/dev-docs/bidders --> 
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
<!-- Describe the change proposed in this pull request -->
First push for new adapter

<!-- For new bidder adapters, please provide the following -->
- test parameters for validating bids
```
{
  bidder: 'my6sense',
  params: {    
    key: 'mW3mLht0kMT',
    pageUrl: '[PAGE_URL]',
    zone: '[ZONE]',
    dataView: ''
    organicClicks:'',
    paidClicks:''
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [x] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

## Other information
<!-- References to related PR or issue #s, @mentions of the person or team responsible for reviewing changes, etc. -->

bretg commented:
Docs PR is https://github.com/prebid/prebid.github.io/pull/846

mike-chowla commented on modules/my6senseBidAdapter.js:
```diff
@@ -0,0 +1,194 @@
+const {registerBidder} = require('../src/adapters/bidderFactory');
+const BIDDER_CODE = 'my6sense';
+const END_POINT = 'http://papi.mynativeplatform.com/pub2/web/hbwidget.json';
```
Adapter does not support https which is required

my6sense commented:
@mike-chowla - we added a version number to the endpoint url and removed the "http" from the url it self and now it starts with // to make sure we support ssl as well.

my6sense commented:
@mike-chowla, we made the changes and pushed them a week ago.
is there anything else we need to do?

mike-chowla commented:
@my6sense Apologies for the delay.  

Question:  I don't see your adapter declaring supportedMediaTypes which I believe means you'll only be bidding manner.  I looked at your doc PR which list supported media type as native.

Are you intending to bid on banner, banner & native, or just native?

my6sense commented:
@mike-chowla, i just added the "supportedMediaTypes" property to the spec object, we are using BANNER & NATIVE types, thanks.

mike-chowla commented:
I'm not getting bids back due to a CORS error.

![screen shot 2018-07-09 at 11 34 16 am](https://user-images.githubusercontent.com/13356910/42469338-40384e46-836c-11e8-9b7d-6e41b27f703e.png)

my6sense commented:
@mike-chowla, i updated our my6senseBidAdapter.md file with a valid widget key "OAJJBW2LRYi2CxfhzqogkA", please re-check it.
thanks
